src/Entity/Carrinho.php

```diff
@@ -16,66 +15,65 @@ class Carrinho
     #[ORM\Column]
     private int $id;
 
-    #[ORM\ManyToOne(inversedBy: 'carrinho', cascade: ['persist', 'remove'])]
-    private Cliente $cliente;
+    #[ORM\ManyToOne(inversedBy: 'carrinhos', cascade: ['persist', 'remove'])]
+    private ?Cliente $cliente = null;
 
     #[ORM\ManyToOne(inversedBy: 'carrinhos')]
-    private User $usuario;
+    private ?User $usuario = null;
 
     #[ORM\Column(length: 255)]
     private string $status;
 
-    #[ORM\Column]
-    private int $valorTotal;
+    #[ORM\Column(nullable: true)]
+    private ?int $valorTotal = null;
 
     #[ORM\Column(type: Types::DATETIME_MUTABLE)]
     private \DateTimeInterface $criadoEm;
 
-    #[ORM\Column(type: Types::DATETIME_MUTABLE,nullable: true)]
+    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable: true)]
     private ?\DateTimeInterface $atualizadoEm = null;
 
-    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable:true)]
+    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable: true)]
     private ?\DateTimeInterface $finalizadoEm = null;
 
     /**
      * @var Collection<int, Item>
      */
     #[ORM\ManyToMany(targetEntity: Item::class, mappedBy: 'carrinhos')]
-    private Collection $items;
+    private Collection $itens;
 
-    public function __construct(Cliente $cliente,User $usuario,string $status = "Aguardando pagamento",int $valorTotal = null)
-    {
+    public function __construct($cliente, $usuario)
+    {   
         $this->cliente = $cliente;
         $this->usuario = $usuario;
-        $this->status = $status;
-        $this->valorTotal = $valorTotal;
+        $this->status = "Pendente";
```

Comment:
para evitar que em alguns lugares usem "Pendente" ou "pendente" ou sei lá "PENDENTE", cria constantes e usa ela sempre que precisar...

```suggestion
        $this->status = "Pendente";
```